strawberryfields/backends/states.py

```diff
@@ -24,7 +24,11 @@
 from scipy.linalg import block_diag
 from scipy.stats import multivariate_normal
 from scipy.special import factorial
-from scipy.integrate import simps
+# support old scipy & new where simps was renamed to simpson
+try:
+    from scipy.integrate import simps
+except:
```

Comment:
```suggestion
except AttributeError:
```
as PEP8 recommends not using a bare `except`, and many linters would catch this.

After this, I'm happy to try to run tests here, but I somewhat suspect you're going to run into the same issues as @lneuhaus in #757  with outdated workflows and additional changes required as part of updates to `numpy`'s API.

Reply:
I completely agree with you. However, I have a fear that the tests have not passed for some time. We shall see.

Reply:
Indeed, now that tests are running, they appear to be complaining about ragged `numpy` arrays. It is unfortunate that the required work for this repo is much larger than the patch applied here for a `scipy` import.